src/main/java/hudson/plugins/git/opt/PreBuildMergeOptions.java

```diff
@@ -80,6 +90,14 @@ public String getRemoteBranchName() {
     }
 
     public boolean doMerge() {
+        return mergeToTarget() || mergeFromSource();
+    }
+
+    public boolean mergeToTarget() {
```

Comment:
It would be better to use <code>isMergeTarget()</code> to follow the Sun/Oracle coding style